### ISSUE TITLE
How to use SourceView?

### ISSUE THREAD

**Issue by seikosantana** [GtkSharp/GtkSharp#313]
I have looked into the samples project on the repository to see examples using the GtkSourceView.

So far what i see is this is sufficient to get syntax highlighting.

```cs
SourceView _textViewCode;
_textViewCode = new SourceView();
_textViewCode.ShowLineNumbers = true;
_textViewCode.Buffer.Language = new LanguageManager().GetLanguage("c-sharp");
_textViewCode.Margin = 3;
scroll2.Child = _textViewCode;
```

So i came up with similar thing:
```cs
    public MainWindow() : base(WindowType.Toplevel)
    {

        SourceView sourceView = new SourceView();

        sourceView.ShowLineNumbers = true;
        sourceView.Margin = 3;
        sourceView.Buffer.Language = new LanguageManager().GetLanguage("c-sharp");
        this.Child = sourceView;
        sourceView.ShowAll();
    }
```
![image](https://user-images.githubusercontent.com/35630407/150304498-c2d11664-0c49-4bfb-bbdd-dba0271af8c7.png)

But i dont get any syntax highlighting. Did I miss something?
Thanks in advance

**Comment by hol430** [comment]
Same code works for me, so likely not a GtkSharp problem. A couple of random thoughts:

- Did you manually install a version of gtksourceview4 (e.g. via msys) or are you using the version of the libs which are automatically downloaded when you build the project?
- Does `new LanguageManager().GetLanguage("c-sharp")` resolve to an actual Language instance, or is it null?
- Have you modified the language manager's search path?
- Finally, a bit of a long shot, but if you've set `sourceView.Buffer.HighlightSyntax = false`, that would also cause this :).

**Comment by seikosantana** [comment]
> Did you manually install a version of gtksourceview4 (e.g. via msys) or are you using the version of the libs which are automatically downloaded when you build the project?

I don't remember doing it so i tried `pacman -Q | grep source` and there's nothing related to sourceview.
I then installed it with `pacman -S mingw-w64-x86_64-gtksourceview4` and it works now.

> Have you modified the language manager's search path?
I haven't tinkered with that at all. All i have is the code above in a class in a gtkapp project.

> Finally, a bit of a long shot, but if you've set sourceView.Buffer.HighlightSyntax = false, that would also cause this :).
I didn't set that

So now that works if i install `mingw-w64-x86_64-gtksourceview4` package.


But the weird part (before I installed that package) is the samples has syntax highlighting while my minimal code has no syntax highlighting. I have GTK and glade installed in msys2 and i don't use the version of the libs that are downloaded.
